packages/client/docusaurus/docs/javascript/02-guides/03-call-and-participant-state.mdx

```diff
@@ -18,48 +24,37 @@ This approach makes it possible to access the state and be notified about change
 Alternatively you can also access the current value of each state variable:
 
 ```typescript
-console.log(call.state.participants);
+const { participants } = call.state;
+console.log(participants);
 ```
 
 It's useful when you need to know the current value, but you don't need to be notified about future changes.
 
 ## Call state
 
-The call state can be accessed by `call.state`.
+The call state can be accessed through `call.state` object as in the examples above.
+In this object, you'll find properties that are reactive and static as well.
+The reactive/observable properties are suffixed with `$` and the static ones are without the suffix.
 
-Here are all the call state properties:
+Here are is an excerpt of the call state properties:
 
 | Reactive value           | Static value            | Description                                                                                                  |
 | ------------------------ | ----------------------- | ------------------------------------------------------------------------------------------------------------ |
 | `members$`               | `members`               | The list of call members                                                                                     |
 | `callingState$`          | `callingState`          | Provides information about the call state. For example, `RINGING`, `JOINED` or `RECONNECTING`.               |
+| `localParticipant$`      | `localParticipant`      | The local participant is the logged-in user.                                                                 |
```

Comment:
These are listed later at participant state

Reply:
Fixed.

Reply:
Thank you